tests/integrations/raftstore/test_merge.rs

```diff
@@ -357,6 +337,7 @@ fn test_node_merge_slow_split_left() {
 fn test_node_merge_slow_split(is_right_derive: bool) {
     let mut cluster = new_node_cluster(0, 3);
     configure_for_merge(&mut cluster);
+    do_not_ensure_all_target_peer_exist(&mut cluster);
```

Comment:
add this to `configure_for_merge`?

Reply:
No. it should be ensured by default. It just used for passing some old tests.